flang/test/Lower/intrinsic-procedures/PRODUCT.f90

```diff
@@ -0,0 +1,103 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
This includes random, random_seed etc. Should they be in a separate file?

Reply:
Moved to a dedicated file.